app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,66 @@
+class LessonsController < ApplicationController
+  def index
+    @lessons = Lesson.where(user_id: current_user.id)
+  end
+
+  def show
+    @lesson = Lesson.find(params[:id])
```

Comment:
báo lỗi

Reply:
dạ cái này phải login mới vào xem bài user làm được

Reply:
```suggestion
    @lesson = Lesson.find(params[:id])
```
bao loi neu k tim thay lesson